modules/install/k3s_master.sh

```diff
@@ -124,7 +124,7 @@ wait_nodes() {
   export PATH="$PATH:/usr/local/bin"
   timeElapsed=0
 
-  while (( timeElapsed < 300 )); do
+  while (( timeElapsed < 1200 )); do
```

Comment:
Which use case does increasing the time here by 4x resolve?

Reply:
A lot of times  in docker and also in Jenkins we have latency that is not allowing the wait and timing out.
We can decrease a bit if you find needed # but less than 600 no.

Reply:
No need to decrease! In some cases it might cause for longer delay when things don't come up properly, but I think that's okay to account for the positive scenario

Reply:
We could potentially get more status output in the log and better test efficiency by using something more like 

```
while ! systemctl is-active k3s; do sleep 2; done
```